integration/airflow/tests/integration/Dockerfile

```diff
@@ -34,11 +34,15 @@ RUN mkdir -p /app && mkdir -p /opt/openlineage
 RUN apt-get update && \
     apt-get install -y python3-dev default-libmysqlclient-dev build-essential
 COPY integration/common /opt/openlineage/integration/common
+COPY integration/sql /app/openlineage/integration/sql
+COPY integration/target /app/openlineage/integration/sql/target
```

Comment:
`COPY integration/target /app/openlineage/integration/sql/target` is already called in base stage.
Also, there's no need to copy whole `integration/sql` as previously built wheel is used.

Reply:
Agreed with not copying. But, we don't import anything from base stage here now; it's effectively a second dockerfile.